src/main/scala/com/microsoft/hyperspace/util/HyperspaceConf.scala

```diff
@@ -48,12 +48,20 @@ object HyperspaceConf {
       .toLong
   }
 
-  def hybridScanDeleteMaxNumFiles(spark: SparkSession): Int = {
+  def hybridScanDeletedRatioThreshold(spark: SparkSession): Float = {
```

Comment:
Shall we change this to `Double`? I am planning to move our conf system to something similar to Spark, and it uses `Double` instead of `Float` for decimal configs.